io.catenax.generic.digital_product_passport/2.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,1209 @@
+   samm:characteristic :SourceCategoryEnumeration ;
+   samm:exampleValue "Legal Information" .
+
+:contentType a samm:Property ;
+   samm:preferredName "Content Type"@en ;
+   samm:description "The type of content which can be expected in the \"content\" property. Examples are a link, restricted link, pdf, excel, etc."@en ;
+   samm:characteristic :SourceEnumeration ;
+   samm:exampleValue "link" .
+
+:header a samm:Property ;
+   samm:preferredName "Header"@en ;
+   samm:description "The header as a short description of the document with a maximum of 100 characters."@en ;
+   samm:characteristic :HeaderTrait ;
+   samm:exampleValue "Sustainability Document Material XY" .
+
+:label a samm:Property ;
+   samm:preferredName "Label"@en ;
+   samm:description "The human readable name of the attribute."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Maximum permitted battery power" .
+
+:description a samm:Property ;
+   samm:preferredName "Description"@en ;
+   samm:description "The description of the attribute context."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Description of an attribute" .
+
+:type a samm:Property ;
+   samm:preferredName "Type"@en ;
+   samm:description "The complex description of the type."@en ;
+   samm:characteristic :TypeCharacteristic .
+
+:data a samm:Property ;
+   samm:preferredName "Data"@en ;
+   samm:description "The content from the attribute which is a depended of the data type and typeUnit."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "23" .
+
+:children a samm:Property ;
+   samm:preferredName "children"@en ;
+   samm:description "Children of the hierarchy."@en ;
+   samm:characteristic :AdditionalDataList .
+
+:StatusEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Status Enumeration"@en ;
+   samm:description "The current status of the product passport declared through either: draft, approved, invalid or expired."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "draft" "approved" "invalid" "expired" ) .
+
+:ValidCharacteristic a samm-c:Either ;
```

Comment:
maybe you could also just model this without Either but xsd:string as datatype. This would make the resulting json schema much easier to process

Reply:
i have several either characterisitcs in this model. It should be either a correct date with the timezone (without time!) or a open string field

Reply:
this seems to be a bug in the json generator. A bug is already created but I think it will take some time until the problem is solved.
https://github.com/eclipse-esmf/esmf-sdk/issues/418

I would suggest to simplify the model and remove the either construct in favor of a pure characteristic with xsd:string datatype